provider/aws/lambda/cfevents/main.go

```diff
@@ -155,3 +133,18 @@ func parseMessage(msg string) (Message, error) {
 
 	return m, nil
 }
+
+func getLogGroup(stack string) (string, error) {
+	res, err := cf.DescribeStackResources(&cloudformation.DescribeStackResourcesInput{
```

Comment:
From experience DescribeStacks seems particularly vulnerable to throttling (we added some caching around this in Rack)

Long term the right answer is for CF/ECS events to go into SQS and we can have a worker pulling data off and making bigger chunks of logs API calls